opm/utility/ECLGraph.cpp

```diff
@@ -1855,6 +1940,69 @@ flux(const PhaseIndex phase) const
     return v;
 }
 
+namespace Opm {
+
+    template <typename T>
+    std::vector<T>
+    ECLGraph::Impl::rawLinearisedCellData(const std::string& vector) const
+    {
+        auto x = std::vector<T>{};  x.reserve(this->numCells());
+
+        for (const auto& G : this->grid_) {
+            const auto xi =
+                G.template activeCellData<T>(*this->src_, vector);
```

Comment:
Sorry about that.  This call does not need the `.template` disambiguation.  Object `G` does not depend on a template parameter.  I'll fix that.